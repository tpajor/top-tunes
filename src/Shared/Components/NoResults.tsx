import * as React from 'react';

const NoResults = () => {
  return (
    <div className='container container--info'>
      <div className='container container--icon-truck' />
      <h4 className='text text--info'>
        No results found
      </h4>
    </div>
  );
};

export default NoResults;
