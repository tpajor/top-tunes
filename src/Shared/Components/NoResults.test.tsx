import * as React from 'react';
import { shallow } from 'enzyme';

import NoResults from './NoResults';

function setup() {

  const wrapper = shallow(<NoResults />);

  return {
    wrapper,
  };
}

describe('NoResults component', () => {
  it('should match snapshot if no change was intended', () => {
    const { wrapper } = setup();

    expect(wrapper).toMatchSnapshot();
  });
});
