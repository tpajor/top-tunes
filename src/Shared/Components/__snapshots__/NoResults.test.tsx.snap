// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component should match snapshot if no change was intended 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NoResults />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="container container--icon-truck"
        />,
        <h4
          className="text text--info"
        >
          No results found
        </h4>,
      ],
      "className": "container container--info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "container container--icon-truck",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "No results found",
          "className": "text text--info",
        },
        "ref": null,
        "rendered": "No results found",
        "type": "h4",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="container container--icon-truck"
          />,
          <h4
            className="text text--info"
          >
            No results found
          </h4>,
        ],
        "className": "container container--info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "container container--icon-truck",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "No results found",
            "className": "text text--info",
          },
          "ref": null,
          "rendered": "No results found",
          "type": "h4",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
